.claude/hooks/post-commit-reset.sh

```
@@ -12,7 +12,8 @@ fi
 
 echo "$COMMAND" | grep -qE 'git\s+commit' || exit 0
 
-STATE_DIR="$(dirname "$0")/state"
+_PROJECT="${CLAUDE_PROJECT_DIR:-$(cd "$(dirname "$0")/../.." && pwd)}"
+STATE_DIR="$_PROJECT/.mercury/state"
 rm -f "$STATE_DIR/review-passed" 2>/dev/null
 git rev-parse HEAD 2>/dev/null > "$STATE_DIR/last-commit-sha" 2>/dev/null
```

Shellcheck (0.11.0)

[error] Multiple redirections compete for stderr. Use cat, tee, or pass filenames instead.

(SC2261)